transcendence-app/src/config/env.validation.ts

```diff
@@ -31,6 +31,14 @@ class EnvironmentVariables {
 
   @IsUrl()
   FORTYTWO_APP_PROFILE_URL!: string;
+
+  @IsString()
+  @IsNotEmpty()
```

Comment:
Could we make this a bit more specific? I wouldn't demand it now, but 'a' something like this would work, and probably it shouldn't.
A bit more specific specifying something like a minimum length, of IsBase64 encoded, or is Hexadecimal, or something like this?

Reply:
Good point. I've changed it to UUIDs. Or we could change to other formats. Please let me know what you think.

Reply:
lgtm